### DISCUSSION TITLE
Introduce yourself!

### DISCUSSION THREAD

**Discussion by Thepigcat76** [MuffinGroup/yet-to-be-named-game#81] [Announcements]
Tell us who you are, share your github profile or just say hi 👋

**Comment by Thepigcat76** [comment]
I am [thepigcat76](https://github.com/Thepigcat76) a developer of the @MuffinGroup core team! And responsible for most of the backend stuff (Nexus lexer, parser, interpreter; Factorization engine). I mainly use rust, kotlin and go. To find out more or chat with me, visit my github profile.

**Comment by N1cohd1** [comment]
Hi, I'm N1co. I have nearly no expiriences with coding (except a little for Java), but I'm a pixel artist who has been designing his own Minecraft texture pack for about 2 years now (procrastination), and I am the Lead Artist of the Muffingroup. I'm also the one who designed most of the textures for ytbng.

**Comment by Trialgamer** [comment]
Hey there! My name @TheHackerChampion . I'm a developer for the MuffinGroup, but actually just a programming beginner. I created our website so for questions you may ask me. If you want to get more information about me, just visit my github profile.
